src/screens/ChannelListScreen/elements/Search.js

```diff
@@ -102,10 +98,9 @@ const styles = StyleSheet.create({
   },
   newPostText: (dynamicColors) => ({
     color: dynamicColors.primary,
-    marginRight: 11,
-    fontFamily: 'Inter-SemiBold',
-    fontSize: 12,
-    lineHeight: 14.52
+    marginRight: dimen.normalizeDimen(4),
+    fontFamily: fonts.inter[600],
+    fontSize: normalizeFontSize(14)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The margin, font family, and font size of the new post text have been changed. Ensure that these changes improve readability and do not cause any layout issues.

```diff
- marginRight: 11,
- fontFamily: 'Inter-SemiBold',
- fontSize: 12,
+ marginRight: dimen.normalizeDimen(4),
+ fontFamily: fonts.inter[600],
+ fontSize: normalizeFontSize(14)
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->